stable/anchore-engine/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: anchore-engine
-version: 1.12.0
+version: 1.12.1
```

Comment:
This is gonna need to be bumped again, another PR snuck in before this one.

Reply:
fixed!